### PR TITLE
Making the little `v` button larger on /questions page

### LINKED ISSUE #10757: Make the little `v` button a little larger on question page
### 💡Format this Issue Into a First Timers Issue
This has been marked as a good candidate for becoming a [first-timers-only issue](https://code.publiclab.org/#r=all) like these, meaning that it's **simple, self-contained**, and with some extra formatting✍️, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider **reformatting it as a first-timers-only issue**, and then ping @publiclab/reviewers to get it labelled🏷. Or, if this is not your first time, **try to solve it yourself**🧑‍💻!

****

### 🤔 What you will need to know.

Make the little `v` button a little larger on a question page like [this one](https://publiclab.org/questions/Softconcept_Africa/09-29-2016/question-put-some-dustduino-in-africa) which is visible on lower screen sizes (mobile, tab).
The relevant code can be found [here](https://github.com/publiclab/plots2/blob/4d31f1d8e4e821d9f148142a50e6e0c105fe174f/app/views/sidebar/_question.html.erb#L5).

https://user-images.githubusercontent.com/78212650/155412993-b4feca44-a594-428a-8029-4cacadef3879.mp4

### PR DESCRIPTION
**Pull request by AbihaFatima**
Fixes #10757  

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

#### Type of Change:
- Code
- User Interface
#### Screenshots:
Before:
patch-3-before

After:
patch-3-after

### CODE REVIEW

**Comment by TildaDares** [comment]
@AbihaFatima This all looks great. Do you think you could add a bit of space before and after the `v` button?  Thanks!

File: app/views/sidebar/_question.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <%= render :partial => "like/like", :locals => {:node => @node, :tagnames => @tagnames } %>
 
-  <a style="margin-top:-16px;margin-bottom:0;" class="btn btn-sm btn-block btn-link d-lg-none" href="javascript:void()" onClick="toggle_sidebar()"><i class="fa fa-chevron-down"></i></a>
+  <a style="margin-top:-16px;margin-bottom:0;" class="btn btn-sm btn-block btn-link d-lg-none" href="javascript:void()" onClick="toggle_sidebar()"><i class="fa fa-chevron-down" style="font-size:26px; padding-top:15px; padding-bottom:15px;"></i></a>
```
Comment (AbihaFatima) [inline app/views/sidebar/_question.html.erb:5]: Here's how it looks:

code-review-changes-1

Comment (TildaDares) [reply app/views/sidebar/_question.html.erb:5]: This looks perfect @AbihaFatima. Thank you!

Comment (TildaDares) [reply app/views/sidebar/_question.html.erb:5]: Also instead of using styles, why not use bootstrap classes? It's better to maintain consistency.

Comment (AbihaFatima) [reply app/views/sidebar/_question.html.erb]: Sure!

**Comment by TildaDares** [comment]
@jywarren Some tests aren't running because of this error:
```
Exit code: 1[28](https://github.com/publiclab/plots2/runs/5556409956?check_suite_focus=true#step:7:28)
Command: git
Arguments: ls-remote --tags --heads git://github.com/jywarren/grow.git
Directory: /home/runner/work/plots2/plots2
Output:
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
Error: Process completed with exit code 128.
```

**Review by TildaDares — APPROVED** [review]
Thanks @AbihaFatima! Great work.

**Pull request merged**